packages/lib/src/vis/scatter/models.ts

```diff
@@ -0,0 +1,7 @@
+import type { NumArray, ScaleType } from '@h5web/shared';
+
+export interface ScatterAxisParams {
+  label?: string;
+  value: NumArray;
+  scaleType?: ScaleType;
+}
```

Comment:
This type is simply `AxisParams` with a required `value`. I could use `extend` but given the few fields, I thought it was clearer to rewrite it.